mysqldsb/service/src/main/java/com/emc/nazgul/dsb/mysql/MySqlDSBMicroService.java

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) [2017] Dell Inc. or its subsidiaries. All Rights Reserved.
+package com.emc.ocopea.dsb.mysql;
+
+import com.emc.microservice.Context;
+import com.emc.microservice.MicroService;
+import com.emc.microservice.MicroServiceInitializationHelper;
+import com.emc.ocopea.hackathon.CloudFoundryClientResourceDescriptor;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Created by liebea on 1/16/17.
+ * Drink responsibly
+ */
+public class MySqlDSBMicroService extends MicroService {
+    private static final String SERVICE_NAME = "MySql DSB";
+    private static final String SERVICE_BASE_URI = "mysql-dsb";
+    private static final String SERVICE_DESCRIPTION = "MySql DSB implementation in java";
```

Comment:
This appears in the API, no need to mention it is in java :) should say something about PCF of p-mysql

Reply:
Good point. Fixed.